components/NewsfeedCard/NewsfeedCardBody.tsx

```diff
@@ -0,0 +1,176 @@
+import CardBootstrap from "react-bootstrap/Card"
+import { useMediaQuery } from "usehooks-ts"
+import { Col, Row } from "../bootstrap"
+
+interface NewsfeedCardBodyProps {
+  imgSrc?: string
+  imgAltTxt?: string
+  text?: string
+  timestamp?: string
+}
+
+export const NewsfeedBillCardBody = (props: NewsfeedCardBodyProps) => {
+  const { imgAltTxt, text, timestamp } = props
+  const isMobile = useMediaQuery("(max-width: 550px)")
+  return (
+    <CardBootstrap.Body className={`p-0`}>
+      <div
+        className={`align-items-center bg-info d-flex mx-2 mt-1 mb-2 p-2 rounded`}
+      >
+        {isMobile ? (
```

Comment:
More of a side note - I think the mobile designs are causing us more pain than they should (and we've seen this across a few PRs now). We should ideally be able to lean on Bootstrap more and set use breakpoint utility classes and column sizes to handle mobile layouts concisely. If we need to keep implementing two designs manually - one for desktop and one for mobile - we're passing up on one of the key advantages of these rendering libraries.

This might need to loop back to a broader discussion with design about standardizing breakpoints and behavior, but IMO it's one that's worth putting our foot down on.